simulator/trainingbot_agent.py

```diff
@@ -22,7 +24,10 @@ def __init__(self, motion_delta=0.5, skew=0.0):
         self.max_force = 1
         self.motion_delta = motion_delta
         self.velocity_limit = 5
+        # As fractions of self.velocity_limit
+        # Range from -1.0 to 1.0
```

Comment:
Does not range [-1.0, 1.0]. It is [-self.velocity_limit, self.velocity_limit]

Reply:
Fixed it in the code, forgot to remove the comment